### DISCUSSION TITLE
Unable to get a page to load from `docs/examples/react-nodejs`

### DISCUSSION THREAD

**Discussion by deven367** [markdoc/markdoc#23] [Q&A] [answered]
Hello, I'm quite new to React. I've been trying to get `markdoc` to work on my system. I cloned the [docs](https://github.com/markdoc/docs) repository. I `cd` into `examples/react-nodejs`, and do a `npm install`. However, when I start the `client` and the `server`, nothing shows up. 

Can anyone help me with this? TIA!
![image](https://user-images.githubusercontent.com/31466137/168198704-1bc240f4-9e0b-4475-b815-4ae5ceb3fb90.png)

**Comment by matv-stripe** [comment]
Hi @deven367 !

Are you running

```
npm run start:client
```

and

```
npm run start:server
```

in different terminal windows? Do either of them have logs that show any errors?

**Reply by deven367** [reply]
Yes, I run both commands in different terminal windows. 
![image](https://user-images.githubusercontent.com/31466137/168445597-069375e3-973a-4bb5-ba5c-e64e711a7c16.png)

**Reply by matv-stripe** [reply]
Interesting, I wonder if this is a windows specific issue. I pushed this change https://github.com/markdoc/docs/pull/57, let me know if it still does not work after that is merged!

**Reply by deven367** [reply]
@matv-stripe I followed the steps as seen in PR #57. It still does not fix the issue.
![image](https://user-images.githubusercontent.com/31466137/168711882-5f88a032-95e7-4187-ada7-a155a662b0a3.png)

**Reply by matv-stripe** [reply]
If you visit http://localhost:3000/variables do you see content? I think we need to improve the sample to support redirection

**Reply by deven367** [reply]
Nothing shows up for `http://localhost:3000/variables` or `http://localhost:3000/home` or `http://localhost:3000/content`.

**Reply by mfix-stripe** [reply]
@deven367 does something show up for you when you open http://localhost:3000/ (with a trailing slash)? 

Which version of Node are you running?

**Reply by deven367** [reply]
I'm running Node v16.14.0
![image](https://user-images.githubusercontent.com/31466137/169187829-9c93200c-4f6d-43d9-8fe6-48cb78d73c18.png)

@mfix-stripe when I try to load `https://localhost:3000/` it redirects me to `https://localhost:3000` and it says `Page not found`
![image](https://user-images.githubusercontent.com/31466137/169188054-82436775-fc6c-4ab1-bba4-2531380b5024.png)

**Reply by mfix-stripe** [reply]
@deven367 if you update this call (https://github.com/markdoc/docs/blob/main/examples/react-nodejs/src/App.js#L11) to hit your server on localhost:4242 directly (instead of using [`proxy`](https://create-react-app.dev/docs/proxying-api-requests-in-development/)), does it work for you then?

**Reply by deven367** [reply]
I tried that. The message being on shown on the screen changes. 
![image](https://user-images.githubusercontent.com/31466137/170805748-a0acb733-cd99-463c-9c8d-6f8a7966bdd5.png)

This is what I see.
![image](https://user-images.githubusercontent.com/31466137/170805758-97de8494-cd0f-487a-b8ad-a4fab434b77c.png)

**Reply by deven367** [reply]
When I add a trailing forward slash at the end, I get a CORS error.

![image](https://user-images.githubusercontent.com/31466137/170805796-8603c472-ad63-4a4b-a203-f90cd04d3ac5.png)

**Comment by mfix-stripe** [comment]
@thecrypticace @arranf @SamuelQuinones — would any of you be able to take a quick look at this and share more details on this issue? I believe this is just a problem with create-react-app for Windows, but it's hard to tell without access to a Windows device. If you can diagnose the problem, pushing [a PR here](https://github.com/markdoc/docs/tree/main/examples/react-nodejs) would be splendid 💫

**Reply by thecrypticace** [reply]
@mfix-stripe I might have some time tomorrow to take a look

**Reply by mfix-stripe** [reply]
Thanks @thecrypticace, no rush whatsoever. I appreciate your help!

**Comment by NotFazed** [comment]
I have this exact same problem.  I'm on Windows, not sure if that is the problem.  3 years after this post, I guess this is still an issue

**Reply by NotFazed** [reply, marked as answer]
Figured it out.  On windows, change this line :

  const files = glob.sync(${ROOT_DIR}/**/*.md);

to

  const files = glob.sync(`${ROOT_DIR}/**/*.md`, { windowsPathsNoEscape: true });
